### PR TITLE
Remaining OpenHands replacements

### PR DESCRIPTION
**Pull request by tobitege**
**What is the problem that this fixes or functionality that this introduces? Does it fix any open issues?**

These should be the final textual replacements for OpenHands (mostly lowercase).

### CODE REVIEW

**Review by neubig — COMMENTED** [review]
I will defer to @xingyaoww and @rbren on this one, I'm not sure what we're doing with `ghcr.io`

**Review by xingyaoww — COMMENTED** [review]
I think our version / docker is not transferred here yet - we probably need to release a new version before we are able to change these? cc @mamoodi

**Comment by rbren** [comment]
These were intentionally omitted, but I'm working on mirroring our old ghcr to the new one. Stay tuend

File: README.md [hunk 1]
```diff
@@ -67,13 +67,13 @@ docker run -it \
     -p 3000:3000 \
     --add-host host.docker.internal:host-gateway \
     --name openhands-app-$(date +%Y%m%d%H%M%S) \
-    ghcr.io/opendevin/opendevin:0.8
+    ghcr.io/openhands/openhands:0.8
```
Comment (rbren) [inline README.md:70]: These should be `all-hands-ai/openhands`

Comment (tobitege) [reply README.md:70]: Thanks, will update now

**Comment by tobitege** [comment]
I hope I got'em all now

**Review by xingyaoww — APPROVED** [review]
LGTM now. We can merge after we got the package migrated

**Comment by tobitege** [comment]
Closing this: in the hectic I created the PR on my `main` branch, which is now blocked.
Will re-issue a PR with the same fixes asap.

**Pull request closed without merging**